lib/qbwc/request/creditmemos.rb

```diff
@@ -0,0 +1,284 @@
+module QBWC
+  module Request
+    class Creditmemos
+      GENERAL_MAPPING = [
+        {qbe_name: "CustomerRef", flowlink_name: "customer_name", is_ref: true},
+        {qbe_name: "ClassRef", flowlink_name: "class_name", is_ref: true},
+        {qbe_name: "ParentRef", flowlink_name: "parent_name", is_ref: true},
+        {qbe_name: "IsTaxIncluded", flowlink_name: "is_tax_included", is_ref: false},
+        {qbe_name: "SalesTaxCodeRef", flowlink_name: "sales_tax_code_name", is_ref: true},
+        {qbe_name: "PONumber", flowlink_name: "po_number", is_ref: false},
+        {qbe_name: "Other", flowlink_name: "other", is_ref: false},
+      ]
+      EXTERNAL_GUID_MAP = [{qbe_name: "ExternalGUID", flowlink_name: "external_guid", is_ref: false, add_only: true}]
```

Comment:
I don't think this is used

Reply:
Nice catch, I'll remove that in the next push